src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using Android.Graphics;
+using AApplication = Android.App.Application;
+
+namespace Microsoft.Maui
+{
+	public class FontManager : IFontManager
+	{
+		readonly ConcurrentDictionary<(string fontFamily, FontAttributes fontAttributes), Typeface?> _typefaces =
+			new ConcurrentDictionary<(string fontFamily, FontAttributes fontAttributes), Typeface?>();
+
+		readonly IFontRegistrar _fontRegistrar;
+
+		Typeface? _defaultTypeface;
+
+		public FontManager(IFontRegistrar fontRegistrar)
```

Comment:
The font manager requires the font registrar because it looks up the font in the registrar, and then opens and caches it here.